File: Bars.Gkh/Entities/BelayOrg/BelayOrganization.cs
namespace Bars.Gkh.Entities
{
    using System;

    using Bars.Gkh.Enums;

    /// <summary>
    /// Страховые организация
    /// </summary>
    public class BelayOrganization : BaseGkhEntity
    {
        /// <summary>
        /// Контрагент
        /// </summary>
        public virtual Contragent Contragent { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Статус
        /// </summary>
        public virtual OrgStateRole OrgStateRole { get; set; }

        /// <summary>
        /// Наименование Контрагента (не хранимое)
        /// </summary>
        public virtual string ContragentName { get; set; }

        /// <summary>
        /// Наименование МО (не хранимое)
        /// </summary>
        public virtual string MuniсipalUnionName { get; set; }

        // Деятельность

        /// <summary>
        /// Описание для деятельности
        /// </summary>
        public virtual string ActivityDescription { get; set; }

        /// <summary>
        /// Основание прекращения деятельности
        /// </summary>
        public virtual GroundsTermination ActivityGroundsTermination { get; set; }

        /// <summary>
        /// Дата прекращения деятельности
        /// </summary>
        public virtual DateTime? DateTermination { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/BelayOrganization.js
Ext.define('B4.model.BelayOrganization', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    requires: [ 'B4.enums.OrgStateRole',
                'B4.enums.GroundsTermination'],
    proxy: {
        type: 'b4proxy',
        controllerName: 'BelayOrganization'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Contragent', defaultValue: null },
        { name: 'OrgStateRole', defaultValue: 10 },
        { name: 'Municipality' },
        { name: 'ContragentName' },
        { name: 'Description' },
        { name: 'ActivityDescription' },
        { name: 'ActivityGroundsTermination', defaultValue: 10 },
        { name: 'DateTermination' },
        { name: 'Inn' }
    ]
});